packages/framework-integration-tests/integration/end-to-end/cart.integration.ts

```diff
@@ -307,22 +307,12 @@ describe('Cart end-to-end tests', () => {
               `,
             })
           },
-          (result) => result?.data?.ProductReadModel?.deleted && result?.data?.ProductReadModel?.id === productId
+          () => true
```

Comment:
What's this `true`? xD

Reply:
@NickSeagull There is a mandatory check for the data returned. I had to return true to get all data from the query and then check that it's null. If I filter by the `id` and the query doesn't find the result, the test automatically fails and returns nothing.